package.json

```diff
@@ -15,9 +15,12 @@
     "uglify-js": "3.3.8"
   },
   "devDependencies": {
-    "standard": "^10.0.0",
+    "fs-extra": "^5.0.0",
```

Comment:
fs-extra needs to be moved to dependencies

All the dependencies that are versioned with `^` need to be versioned with `~` instead